### PR TITLE
Package multicont.1.0.0

### PR DESCRIPTION
**Pull request by dhil**
### `multicont.1.0.0`
Multi-shot continuations in OCaml
This library provides facilities for programming with multi-shot continuations in OCaml

---
* Homepage: https://github.com/dhil/ocaml-multicont
* Source repo: git+https://github.com/dhil/ocaml-multicont.git
* Bug tracker: https://github.com/dhil/ocaml-multicont

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
This package fails to build on bytecode-only platforms (x86, arm, ppc64, s390x):
```
#=== ERROR while compiling multicont.1.0.0 ====================================#
# context              2.2.0~alpha~dev | linux/x86_32 | ocaml-base-compiler.5.0.0 | pinned(https://github.com/dhil/ocaml-multicont/archive/refs/tags/v1.0.0.tar.gz)
# path                 ~/.opam/5.0/.opam-switch/build/multicont.1.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build make all
# exit-code            2
# env-file             ~/.opam/log/multicont-7-bde193.env
# output-file          ~/.opam/log/multicont-7-bde193.out
### output ###
# ocamlc -c -strict-formats -strict-sequence -safe-string -bin-annot -warn-error -a multicont.mli multicont.ml
# ocamlc -c fiber_primitives.c
# ocamlc -c multicont_stubs.c
# ocamlmklib -oc multicont -dllpath . fiber_primitives.o multicont_stubs.o
# ocamlmklib -o multicont -oc multicont -dllpath . multicont.cmo
# ocamlopt -c -strict-formats -strict-sequence -safe-string -bin-annot -warn-error -a multicont.mli multicont.ml
# make: ocamlopt: No such file or directory
# make: *** [Makefile:48: multicont.cmx] Error 127
```
Do you want to try to fix this or mark it as incompatible with bytecode-only mode?

**Comment by dhil** [comment]
Aha! I should invoke `make byte` for bytecode-only mode; currently I invoke `make all` irrespective of the mode. Would I be able to use the filter `{ !ocaml:native }` to reliably detect bytecode-only mode?

E.g.
```opam
build: [
 [ make `byte` { !ocaml:native } ]
 [ make `all` { ocaml:native } ]
]
```

(I assume here that the bytecode compiler is available whenever the native mode is toggled; is this assumption true?)

File: packages/multicont/multicont.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+authors: "Daniel Hillerström <daniel.hillerstrom@ed.ac.uk>"
+synopsis: "Multi-shot continuations in OCaml"
+description: "This library provides facilities for programming with multi-shot continuations in OCaml"
+homepage: "https://github.com/dhil/ocaml-multicont"
+dev-repo: "git+https://github.com/dhil/ocaml-multicont.git"
+bug-reports: "https://github.com/dhil/ocaml-multicont"
+license: "MIT"
+
+build: [
+  [ make "all" ]
```
Comment (mseri) [inline packages/multicont/multicont.1.0.0/opam:12]:
```suggestion
  [
    make
    "byte" {!ocaml:native}
    "all" {ocaml:native}
  ]
```

**Comment by dhil** [comment]
Excellent! I will update the PR later today. I need to tinker a little with the Makefile to support bytecode-only installation (unless dune/build systems don't care about missing native libs).

**Comment by mseri** [comment]
Unless you plan to modify the released tarball, which would not be needed for just this change, this could be merged. Let me know

**Comment by dhil** [comment]
Alright! Happy to merge as-is. I will change my OPAM file accordingly. Thanks for the help!

**Pull request merged**